tests/Benchmarks/Benchmarks.csproj

```diff
@@ -16,6 +16,11 @@
   <ItemGroup>
     <EmbeddedResource Include="System.Text.Primitives\SampleTexts\*.txt" />
   </ItemGroup>
+  <ItemGroup>
+    <Compile Include="..\System.Buffers.Primitives.Tests\BufferSegment.cs" Link="BufferSegment.cs" />
```

Comment:
Include a project reference to the entire System.Buffers.Primitives.Tests instead, similar to other project references?

https://github.com/dotnet/corefxlab/pull/2419/files#diff-2d9cd050a80433427e0c8791e9b28ed5R51

Reply:
Tried that first, it didn't work out. We link in those files in multiple test projects. I should probably move the helpers to a shared project.

Reply:
> Tried that first, it didn't work out.

What ended up breaking?

Also, I tried to run the Reader_Binary tests locally, and i can't. I am getting the following errors for some reason. Have you come across this before @JeremyKuhne?
```text
// Validating benchmarks:
Only 1 benchmark method in a group can have "Baseline = true" applied to it, group DefaultJob-[Length=99, CodePointInfo=Syste(...)Point [43]] in class EncodeFromUtf16toUtf8 has 7
Only 1 benchmark method in a group can have "Baseline = true" applied to it, group DefaultJob-[Length=999, CodePointInfo=Syste(...)Point [43]] in class EncodeFromUtf16toUtf8 has 7
Only 1 benchmark method in a group can have "Baseline = true" applied to it, group DefaultJob-[Length=9999, CodePointInfo=Syste(...)Point [43]] in class EncodeFromUtf16toUtf8 has 7
Only 1 benchmark method in a group can have "Baseline = true" applied to it, group DefaultJob-[Length=99, CodePointInfo=Syste(...)Point [43]] in class EncodeFromUtf32toUtf16 has 7
Only 1 benchmark method in a group can have "Baseline = true" applied to it, group DefaultJob-[Length=999, CodePointInfo=Syste(...)Point [43]] in class EncodeFromUtf32toUtf16 has 7
Only 1 benchmark method in a group can have "Baseline = true" applied to it, group DefaultJob-[Length=9999, CodePointInfo=Syste(...)Point [43]] in class EncodeFromUtf32toUtf16 has 7
Only 1 benchmark method in a group can have "Baseline = true" applied to it, group DefaultJob-[Length=99, CodePointInfo=Syste(...)Point [43]] in class EncodeFromUtf8toUtf16 has 7
Only 1 benchmark method in a group can have "Baseline = true" applied to it, group DefaultJob-[Length=999, CodePointInfo=Syste(...)Point [43]] in class EncodeFromUtf8toUtf16 has 7
Only 1 benchmark method in a group can have "Baseline = true" applied to it, group DefaultJob-[Length=9999, CodePointInfo=Syste(...)Point [43]] in class EncodeFromUtf8toUtf16 has 7
```

Any ideas? cc @adamsitnik